api/transport/propagation.go

```diff
@@ -112,10 +118,11 @@ func (e *ExtractOpenTracingSpan) Do(
 
 // UpdateSpanWithErr sets the error tag on a span, if an error is given.
 // Returns the given error
-func UpdateSpanWithErr(span opentracing.Span, err error) error {
+func UpdateSpanWithErr(span opentracing.Span, err error, errCode yarpcerrors.Code) error {
```

Comment:
The function is actually used internally in [several places](https://sg.uberinternal.com/search?q=context:global+repo:%5Ecode%5C.uber%5C.internal/uber-code/go-code%24+transport.UpdateSpanWithErr&patternType=keyword&sm=0), so it will be a breaking change when changing the function signature.

Reply:
thanks, updated to make errCode params optional